03/index.html

```diff
@@ -1,16 +1,57 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #03</title>
+    <link rel="stylesheet" href="./styles/global.css">
 </head>
+
 <body>
-    
-    <main class="t">T</main>
-    <header class="p">P</header>
-    <side class="v">V</side>
+    <div class="grid-wrapper">
+        <main class="t"> Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos quia eligendi quo quis cum
+            quae pariatur harum laborum reiciendis doloribus tenetur voluptatem delectus voluptate ipsa consectetur
+            iure, et incidunt accusantium ullam soluta recusandae quam enim. Harum amet sunt culpa saepe sit tempora
+            adipisci, officiis perspiciatis. Quibusdam quis eos dignissimos, iusto quam dolor a tempora. Facilis est
+            exercitationem commodi sunt id voluptatum obcaecati ea aliquid sapiente incidunt ducimus, minus
+            perspiciatis, assumenda sint nihil aperiam neque rem ex? Laudantium quibusdam aut incidunt ipsam quam, culpa
+            reprehenderit neque facere mollitia quod! Sed, debitis reprehenderit recusandae atque mollitia, deserunt
+            consequuntur eaque eius et quae cupiditate veniam omnis commodi libero corrupti sequi obcaecati, architecto
+            minus fugiat. Harum velit iusto nulla nesciunt iure quam dolor ullam eos placeat! Voluptas, repellendus
+            adipisci eos nesciunt laboriosam magni exercitationem debitis voluptatibus temporibus officiis, quaerat
+            labore iure repellat eveniet a necessitatibus totam veniam rerum. Nemo, explicabo velit? Eligendi, iusto.
+            Deleniti cumque repellat id? Cumque cupiditate quasi vero, qui officiis quis commodi ab minus impedit
+            laboriosam reprehenderit dolores, ipsa perferendis modi recusandae id dolor incidunt repellat voluptates
+            blanditiis et ad nesciunt asperiores? Aut sed temporibus aliquid officia soluta labore deleniti magnam?
+            Commodi rerum accusamus quos a? Ipsum asperiores consequuntur debitis illum?</main>
+        <header class="p">
+            <picture>
+                <source
+                    media="(min-width: 1200px)"
+                    srcset="img/coneflower-8168628_1920.jpg"
+                >
+                <source
+                    media="(min-width: 600px)"
+                    srcset="img/coneflower-8168628_640.jpg"
+                >
+                <source srcset="img/coneflower-8168628_640.jpg">
+                <img
+                    src="img/coneflower-8168628_1920.jpg"
+                    alt="blooming flowers"
+                />
+            </picture>
+
+
+        </header>
+        <side class="v">
+       <iframe src="https://youtu.be/o1tj2zJ2Wvg" frameborder="0"
```

Comment:
Udostępniając film musisz użyć innego adresu np.: https://www.youtube.com/embed/o1tj2zJ2Wvg
Niestety ten film nie może być udostępniony na stronie:
![image](https://github.com/devmentor-pl/practice-html-and-css-rwd/assets/55095257/c61b1f19-7c89-42c8-aa0e-c6f31b8e621d)